# AttCovEuler block
# ROS message header
std_msgs/Header header

# SBF block header including time header
BlockHeader  block_header   

uint8        error   
float32      cov_headhead
float32      cov_pitchpitch
float32      cov_rollroll
float32      cov_headpitch
float32      cov_headroll
float32      cov_pitchroll
